packages/utils/src/common/sharding.ts

```diff
@@ -66,7 +66,8 @@ export const shardInfoToPubsubTopics = (
     // Autosharding: single shard from application and version
     return [
       contentTopicToPubsubTopic(
-        `/${shardInfo.application}/${shardInfo.version}/default/default`
+        `/${shardInfo.application}/${shardInfo.version}/default/default`,
+        shardInfo.clusterId
```

Comment:
I think we need this fix [here](https://github.com/waku-org/js-waku/issues/1902) because Autosharding is possible on other clusterIds